sorc/ncep_post.fd/SURFCE.f

```diff
@@ -2650,12 +2697,16 @@ SUBROUTINE SURFCE
          if(grib=='grib2') then 
            cfld=cfld+1
            fld_info(cfld)%ifld=IAVBLFLD(IGET(783))
-           if (ifhr==0) then 
+           fld_info(cfld)%ntrange=1
+           IF (MODELNAME == 'FV3R' .OR. MODELNAME == 'RAPR') THEN 
```

Comment:
@KarinaAsmar-NOAA It turns out that RRFS doesn't use this block to output their products. You may remove the setting "MODELNAME=='FV3R' ".